firmware/qemu/src/bin/hints.rs

```diff
@@ -17,6 +17,8 @@ fn main() -> ! {
     defmt::info!("hex alt {=u8:#x}", x);
     defmt::info!("HEX     {=u8:X}", x);
     defmt::info!("HEX alt {=u8:#X}", x);
+    defmt::info!("octal   {=u8:o}", x);
```

Comment:
Could we add the tests at the end, to minimise the diff on the expected output file?

Reply:
Oh, actually, that probably isn't going to help. Never mind.